1-org/envs/shared/projects.tf

```diff
@@ -168,7 +168,7 @@ module "interconnect" {
   random_project_id        = true
   random_project_id_length = 4
   default_service_account  = "deprivilege"
-  name                     = "${local.project_prefix}-c-interconnect"
+  name                     = "${local.project_prefix}-net-interconnect"
```

Comment:
I understand a large portion of these label changes are already in main - I noticed them when I was doing an upstream sync last sunday
specific example
net-interconnect for 171
https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/1-org/envs/shared/projects.tf#L171

also minor and not really functionality affecting but there is a pending PR on 2 changes that look to have been missed in the last PR on this yaml
https://github.com/terraform-google-modules/terraform-example-foundation/pull/1232/files
for #1231

Reply:
I think we are good - I was using a 3hr old diff view - 1-org is out of the patch now
/lgtm